boa/src/object/mod.rs

```diff
@@ -592,6 +592,11 @@ impl Object {
         }
     }
 
+    #[inline]
+    pub fn expect_map_ref(&self) -> &OrderedMap<JsValue> {
```

Comment:
I'm not a fan of this. It makes things less explicit, and only removes a few characters. It adds more functions and panics are much more difficult to debug.

At the minimum, they should have the `#[track_caller]` attribute.

Reply:
Yeah, maybe that's not a great idea. Hm, I think I can modify the code to avoid expects or explicitly use `expect` where necessary.